certified-connectors/UiPath Orchestrator/apiProperties.json

```diff
@@ -0,0 +1,85 @@
+{
+  "properties":
+  {
+    "connectionParameters":
+    {
+      "token":
+      {
+        "type": "oauthSetting",
+        "oAuthSettings":
+        {
+          "identityProvider": "oauth2",
+          "clientId": "[enter client id]",
+          "scopes":
+          [
+            "OR.Execution.Read OR.Folders.Read OR.Jobs.Write OR.Queues offline_access"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties":
+          {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters":
+          {
+            "authorizationUrl":
+            {
+              "value": "https://cloud.uipath.com/identity_/connect/authorize"
+            },
+            "tokenUrl":
+            {
+              "value": "https://cloud.uipath.com/identity_/connect/token"
+            },
+            "refreshUrl":
+            {
+              "value": "https://cloud.uipath.com/identity_/connect/token"
+            }
+          }
+        }
+      },
+      "organizationName":
+      {
+        "type": "string",
+        "uiDefinition":
+        {
+          "constraints":
+          {
+            "required": "true"
+          },
+          "description": "Specify your organization name.",
+          "displayName": "Organization Name",
+          "tooltip": "Specify your organization name."
+        }
+      },
+      "tenantName":
+      {
+        "type": "string",
+        "uiDefinition":
+        {
+          "constraints":
+          {
+            "required": "true"
+          },
+          "description": "Specify your tenant name.",
+          "displayName": "Tenant Name",
+          "tooltip": "Specify your tenant name."
+        }
+      }
+    },
+    "iconBrandColor": "#D1232B",
+    "capabilities":[],
+    "policyTemplateInstances":
+    [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Route to Tenant Address",
+        "parameters":
+        {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://cloud.uipath.com/@connectionParameters('organizationName')/@connectionParameters('tenantName')"
+        }
+      }
+    ],
+    "publisher": "UiPath"
```

Comment:
@TuckermanRav please add stackOwned property here, you can see information about this property in [this template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json).